--- origsrc/xfce4-dev-tools-4.4.0/m4macros/xdt-python.m4	2007-01-20 03:29:09.000000000 -0600
+++ src/xfce4-dev-tools-4.4.0/m4macros/xdt-python.m4	2007-05-06 21:44:28.468750000 -0500
@@ -51,16 +51,20 @@
   case "${xdt_python_OSNAME}" in
     nt)
       xdt_python_INCLUDES="include"
+      xdt_python_LIBS="lib"
       ;;
     *)
       xdt_python_INCLUDES="include/python${PYTHON_VERSION}"
+      xdt_python_LIBS="lib/python${PYTHON_VERSION}/config"
       ;;
   esac
   PYTHON_INCLUDES="-I${xdt_python_PREFIX}/${xdt_python_INCLUDES}"
   if test x"${xdt_python_EPREFIX}" != x"${xdt_python_PREFIX}"; then
     PYTHON_INCLUDES="${PYTHON_INCLUDES} -I${xdt_python_EPREFIX}/${xdt_python_INCLUDES}"
   fi
+  PYTHON_LIBS="-L${xdt_python_EPREFIX}/${xdt_python_LIBS} -lpython${PYTHON_VERSION}"
   AC_SUBST([PYTHON_INCLUDES])
+  AC_SUBST([PYTHON_LIBS])
 
   dnl now check if the headers exists
   save_CPPFLAGS="$CPPFLAGS"
